resfit/fit_resonator/fit_S_data.py

```diff
@@ -777,7 +777,44 @@ def monte_carlo_fit(xdata= None,ydata=None,parameter=None,Method = None):
         stop_MC = True
     return parameter,stop_MC, error
 
-
+def get_header(line: str):
+    if (line.strip()[0:1]!='#'):
+        print("Header not found in file.")
+        quit()
+    line = line[2:]
+    if line[0:2].lower()=='hz':
+        frequency_units=line[0:2].lower()
+        line = line[3:]
+    elif line[0:3].lower()=='khz':
+        frequency_units=line[0:3].lower()
+        line = line[4:]
+    elif line[0:3].lower()=='mhz':
+        frequency_units=line[0:3].lower()
+        line = line[4:]
+    elif line[0:3].lower()=='ghz':
+        frequency_units=line[0:3].lower()
+        line = line[4:]
+    else:
+        print('Frequency units not found')
+        quit()
+    if line[0:1].lower()!='s':
+        print('Data is not an S parameter measurement')
+        quit()
+    line = line[2:]
+
+    if line[0:2].lower()=='db':
+        data_format=line[0:2].lower()
+        line = line[3:]
+    elif line[0:2].lower()=='ma':
+        data_format=line[0:2].lower()
+        line = line[3:]
+    elif line[0:2].lower()=='ri':
+        data_format=line[0:2].lower()
+        line = line[3:]
+    else:
+        print('Data format not found')
+        quit()
+    return frequency_units, data_format
```

Comment:
this is fine for now, but it would be nice to make an [enum](https://docs.python.org/3/library/enum.html) out of frequency units and data_format